angular-client/src/data-type.enum.ts

```diff
@@ -0,0 +1,74 @@
+export enum DataTypeEnum {
+  DRIVER = 'Driver',
+  LOCATION = 'location',
+  POINTS = 'GPS/Location',
+  VIEWERS = 'Viewers',
+
+  // Special Latency info sent by Scylla
+  LATENCY = 'Old_Latency',
+  NEW_LATENCY = 'Latency',
+
+  //Fake Data Points
+  MOTOR_USAGE = 'Motor Usage',
+  COOL_USAGE = 'Cooling Usage',
+  STEERING_ANGLE = 'Steering Angle',
+  TORQUE = 'Torque',
+  BRAKE_PRESSURE = 'Brake Pressure',
+  ACCELERATION = 'Acceleration',
+  XYZAccel = 'XYZAcceleration',
```

Comment:
@bracyw technically some of this info is being made available, but it's not a simple single datapoint, wanna figure this out rq in this PR or wait?

Reply:
Eh let's wait it's pretty wierd. Basically there are 4 accelerations.